README.md

```diff
@@ -36,15 +36,16 @@ GF_INSTALL_PLUGINS=grafana-pyroscope-app
 ## Getting Started
 
 1. In the main navigation bar, click on Explore > Profiles.
-2. You’ll land in the service overview page that shows time series and CPU utilization visualizations for all the services in your selected Pyroscope instance.
-3. Change your data source with the drop-down on the top left.
+2. You’ll land in the "All services" overview that shows time series and CPU utilization visualizations for all the services in your selected Pyroscope instance.
+3. If needed, change your data source with the drop-down on the top left.
 4. Modify your time range in two ways:
 
    - Use the standard time range picker on the top right.
    - Click and drag the time range you want to see on any time series visualization.
 
-5. Select the service you would like to explore. This takes you to the Service page.
-6. Filter profiles based on labels.
+5. Select the service you would like to explore by clicking on "Profile types" on any time series visualization. This takes you to the "Profile types" overview for that service.
+6. Select the profile type you would like to explore by clicking on "Flame graph" on any time series visualization.
+7. View the flame graph.
```

Comment:
Changing the UI elements to be bolded and changing clicking on to selecting, in case someone is using a smaller screen like a tablet. 
```suggestion
5. Select the service you would like to explore by selecting **Profile types** on any time series visualization. This takes you to the **Profile types** overview for that service.
6. Select the profile type you would like to explore by selecting **Flame graph** on any time series visualization.
7. View the flame graph.
```